### PR TITLE
Structured repository for BT nodes

### PR DESCRIPTION
**Pull request by rodperex**
Structure based on the defined taxonomy. One package per category

### CODE REVIEW

File: bt_nodes/perception/include/perception/IsDetectedPro.hpp [hunk 1]
```diff
@@ -0,0 +1,79 @@
+// Copyright 2021 Intelligent Robotics Lab
```
Comment (Juancams) [inline bt_nodes/perception/include/perception/IsDetectedPro.hpp:1]: please correct the year ^^

File: bt_nodes/perception/src/IsDetectedPro.cpp [hunk 2]
```diff
@@ -0,0 +1,165 @@
+// Copyright 2021 Intelligent Robotics Lab
```
Comment (Juancams) [inline bt_nodes/perception/src/IsDetectedPro.cpp:1]: same

Comment (rodperex) [reply bt_nodes/perception/src/IsDetectedPro.cpp]: solved

**Comment by rodperex** [comment]
Requested changes applied

**Pull request merged**